eradiate/scenes/surface/_centralpatch.py

```diff
@@ -0,0 +1,129 @@
+import typing as t
+
+import attr
+import pint
+import pinttr
+
+import eradiate
+from ._core import Surface, surface_factory
+from ._lambertian import LambertianSurface
+from ..core import KernelDict
+from ... import converters, validators
+from ..._util import onedict_value
+from ...attrs import documented, parse_docs, AUTO, AutoType
+from ...contexts import KernelDictContext
+from ...units import unit_context_config as ucc
+
+
+@surface_factory.register(type_id="central_patch")
+@parse_docs
+@attr.s
+class CentralPatchSurface(Surface):
+    """
+    Central patch surface element [``central_patch``]
+
+    This class creates a square surface to which two BSDFs will be attached.
+
+    The central patch will always be located in the center of the surface.
+    """
+
+    central_patch: Surface = documented(
+        attr.ib(
+            factory=LambertianSurface,
+            converter=surface_factory.convert,
+            validator=attr.validators.instance_of(Surface),
+        ),
+        doc="Inner surface specification. "
+        "This parameter can be specified as a dictionary which will be "
+        "interpreted by "
+        ":meth:`SurfaceFactory.convert() <.SurfaceFactory.convert>`.",
+        type=":class:`.Surface`",
+        init_type=":class:`.Surface` or dict",
+        default=":class:`LambertianSurface() <.LambertianSurface>`",
+    )
+
+    outer_surface: Surface = documented(
```

Comment:
How about `background_surface` instead, to clearly emphasise the idea of a background on which a patch is applied.